scripts/metrics/common/logging.py

```diff
@@ -0,0 +1,65 @@
+"""
```

Comment:
Why does this metrics tool need a custom logging setup managed in its own module? Why not configure with logging.basicConfig directly in the metrics Main?

Reply:
Scalability/standardization if other metrics scripts will be separate from the current